test/functional/legacy/increment_spec.lua

```diff
@@ -720,4 +739,10 @@ describe('Ctrl-A/Ctrl-X on visual selections', function()
       eq({}, nvim.get_vvar('errors'))
     end)
   end
+
+  it('does not drop leading zeroes', function()
+    execute('set nrformats&vi') -- &vi makes Vim compatible
```

Comment:
What I meant is it doesn't exist in the original test script in this function.  It's in the SetUp function, which is analogous to busted's before_each.

That syntax is explained in [`:help :set-default`](http://vimhelp.appspot.com/options.txt.html#:set-default)